tpr/backoff.go

```diff
@@ -0,0 +1,25 @@
+package tpr
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
```

Comment:
When you think this is cool, fine with me. I only do not see why the metric "max retries" matters. The time based default the lib provides would be good enough for me. Up to you.